CHANGELOG.md

```diff
@@ -2,9 +2,44 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.9 September 22, 2021
+# 10.0.0 October 6, 2021
 
-**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+**The Mapbox Maps SDK for Android is now ready for production use.**
+
+## Breaking changes ⚠️
+* Add `@JvmOverloads` where applicable to provide better experience for Java users. ([#656](https://github.com/mapbox/mapbox-maps-android/pull/656))
+* Refactor gestures configuration options to be aligned better across platforms. ([#672](https://github.com/mapbox/mapbox-maps-android/pull/672))
+* Apply geojson data (using data, url, feature, featureCollection, geometry functions) is fully async now. ([#699](https://github.com/mapbox/mapbox-maps-android/pull/699))
+* Update `getLayerAs` function to return nullable Layer type. ([#673](https://github.com/mapbox/mapbox-maps-android/pull/673))
+* Update map events data models. ([#712](https://github.com/mapbox/mapbox-maps-android/pull/712))
+* Refactor MapEvents listeners, so that each listener will include one event data property. ([#718](https://github.com/mapbox/mapbox-maps-android/pull/718))
```

Comment:
same for these two PRs, we have a more detailed list of changes, would be good to include them to at least in release notes.